style.css

```diff
@@ -3,490 +3,355 @@
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-}
-
-textarea {
-  width: 100%;
-  border-radius: 8px;
-  border: #fff;
-  padding: 10px 16px;
-  box-sizing: border-box;
-}
-
-input[type=submit] {
-  width: 50%;
-  float: left;
-  padding: 12px 16px;
-  background-color: #fff;
-  border: #fff;
-  border-radius: 8px;
-  box-sizing: border-box;
-  color: #6070ff;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  letter-spacing: 3%;
-  margin-top: 22px;
-}
-
-.contact-area {
-  background-color: #fff;
-  padding: 16px 16px 16px 0;
-}
-
-.contact-desc {
-  color: #fff;
-  height: auto;
-}
-
-.contact-title {
-  color: #fff;
-}
-
-.see-demo-btn {
-  border-radius: 8px;
-  background-color: #fff;
-  padding: 12px;
-  border: 1px solid #6070ff;
-  color: #396df2;
-  font-size: 17px;
-  line-height: 24px;
-  font-weight: 500;
-  letter-spacing: 3%;
-  margin-top: 24px;
-  margin-bottom: 28px;
-}
-
-.see-demo-btn:hover {
-  background-color: #6070ff;
-  border-radius: 8px;
-  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  color: #fff;
-  transition: 0.76s;
-}
-
-.see-demo-btn:active {
-  background: #2230d2;
-  border-radius: 8px;
-  color: #fff;
-}
-
-.see-demo-btn:disabled {
-  color: #5e6c84;
-  background-color: #fff;
-  border: 1px solid #c1c7d0;
-}
-
-.see-demo-btn:focus {
-  color: #fff;
-  background-color: #4053fc;
-  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  border-radius: 8px;
+@media screen and (min-width: 768px) {
+  .nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 29px 20px;
+  }
+
+  .logo {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    text-decoration: none;
+    margin: 10px 12px;
+  }
+
+  .nav-items li {
+    list-style-type: none;
+    display: inline;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    padding: 10px 30px 12px 12px;
+  }
+
+  .title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    padding: 150px 170px 20px 90px;
+  }
+
+  .paragraph {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    padding-left: 90px;
+  }
+
+  .connect {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    padding-left: 90px;
+    padding-top: 20px;
+  }
+
+  .icons li {
+    padding-left: 90px;
+    padding-top: 10px;
+  }
+
+  .introduction {
+    background-image: url("imgs/Header\ bg.png");
+    background-repeat: no-repeat;
+    border-radius: 0 0 0 60px;
+    padding: 30px;
+  }
```

Comment:
- [ ] Kindly complete your home page behind the social icons to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193379080-a9b25896-47a8-44b6-a63c-0e9378d93d2e.png) | ![image](https://user-images.githubusercontent.com/58642949/193379097-2bc50cf7-c648-4eee-aa49-c0c9c31332c4.png) |